tests/Jenkins/utilities.sh

```diff
@@ -345,7 +345,7 @@ function installDIRAC(){
     } >> "$CLIENTINSTALLDIR/dirac-ci-install.cfg"
   fi
 
-  if ! ./dirac-install -r $DIRAC_RELEASE -t client $INSTALLOPTIONS "$CLIENTINSTALLDIR/dirac-ci-install.cfg" $DEBUG; then
+  if ! ./dirac-install -r $DIRAC_RELEASE -t client "${INSTALLOPTIONS[@]}" "$CLIENTINSTALLDIR/dirac-ci-install.cfg"; then
```

Comment:
Why did you remove the `$DEBUG` flag?

Reply:
because it is already in the `INSTALLOPTIONS` array